README.md

```diff
@@ -80,6 +80,43 @@ qunitHarness
     .catch(function (err) {
         console.log(err);
     });
+
+//Testing in local environment
+//Browser opening mechanism uses the "open" method from "testcafe-browser-tools" library.
+//For more information about "browserInfo" options, see https://github.com/DevExpress/testcafe-browser-tools/blob/b11784e722b370de55473ca91f32373b95821bab/API.md#open 
```

Comment:
`https://github.com/DevExpress/testcafe-browser-tools/blob/master/API.md#open` I guess this link is better

Reply:
You can't place a link inside of a code blocks.
https://gist.github.com/kirovboris/ec6984e88182c9c978e17b9fbc1a0c8e

Reply:
I'm about the content of the link. Replace `b11784e722b370de55473ca91f32373b95821bab` with `master`

Reply:
Sure